.gitattributes

```diff
@@ -26,3 +26,4 @@
 *.otf filter=lfs diff=lfs merge=lfs -text
 *.shape filter=lfs diff=lfs merge=lfs -text
 *.jpeg filter=lfs diff=lfs merge=lfs -text
+assets/textures/MembraneNormals.png filter=lfs diff=lfs merge=lfs -text
```

Comment:
This should not be needed. All *.png files should be already included in here

Reply:
More precisely I meant that this file already contains line 9 which is: `*.png filter=lfs diff=lfs merge=lfs -text` so this line should be entirely redundant.

Reply:
> More precisely I meant that this file already contains line 9 which is: `*.png filter=lfs diff=lfs merge=lfs -text` so this line should be entirely redundant.

alright i'll delete that line real quick